web/pages/group/[...slugs]/index.tsx

```diff
@@ -108,18 +109,27 @@ export async function getStaticPropz(props: { params: { slugs: string[] } }) {
   }
 }
 
-function toTopUsers(userScores: { [userId: string]: number }, users: User[]) {
+function toTopUsers(
+  userScores: { [userId: string]: number },
+  userIds: string[]
+) {
+  // sort userScores by score and trim to top 50
+  const sortedUserScores = Object.fromEntries(
+    Object.entries(userScores)
+      .sort(([, scoreA], [, scoreB]) => scoreB - scoreA)
+      .slice(0, 50)
+  ) as { [userId: string]: number }
+
```

Comment:
Why do we need to sort these, when we sort their entries below? Seems like you could remove this code.